manual.asc

```diff
@@ -1052,6 +1052,13 @@ To stop receiving notifications go to Deck options > Reminders and unmark the ch
 
 Notifications only work for top level decks. Please let us know if you want us to add notifications for subdecks too.
 
+[[setlanguagehint]]
+=== Automatic Language Selection
+AnkiDroid has an in Automatic Language Selection feature in the Note Editor starting with AnkiDroid 2.13. This feature allows you to define a default language to be used in your keyboard for a field in a note type.
+
+For example, if you have Russian and English note fields, the keyboard layout (in keyboards like SwiftKey and Swype) will switch to Russian in the first field and back to English in the second automatically when the fields get focus. https://youtu.be/JrxDjTrRhBE[Checkout video reference for demonstration]
```

Comment:
> in keyboards like SwiftKey and Swype

The last time that I checked, SwiftKey doesn't support `setImeHintLocales`

https://support.swiftkey.com/hc/en-us/community/posts/360055542372-SwiftKey-does-not-switch-setImeHintLocales- (2019)

Swype's been pulled from the Play Store.

Reply:
Haha - that's amusing, both examples as fails. I guess the lesson is: do not give examples by name in documentation, simply state the requirement they must meet ("that support setImeHintLocales", the standard keyboard language hint system in Android")

Reply:
@avinashbest can you propose a quick follow-up PR that removes the named keyboard apps and just states they need to support the setImeHintLocales Android API ?

Reply:
Sure, Why not